src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -103,12 +103,25 @@ fi
 	echo $APP_PACKAGES_PATH > $SITE_PACKAGES_PATH"/oryx.pth"
 {{ end }}
 
+packageWheelType={{ PythonPackageWheelProperty }}
```

Comment:
It doesn't look like `packageWheelType` is used -- am I missing something? ☺️

Reply:
no it was a debug piece .. will remove it